mitmproxy/flowfilter.py

```diff
@@ -381,6 +381,17 @@ def __call__(self, f):
         return f.server_conn.address and self.re.search(r)
 
 
+class FDstIP(_Rex):
+    code = "ip"
```

Comment:
Is there a reason why we can't use `~dst`? It feels like that could be good enough.I would like to avoid extending the filter syntax unless there's an urgent need. :)

Reply:
The following is the code for ~dst.
This uses f.server_conn.address for filtering, but f.server_conn.address isn't IP address. It's host name.
So I need to add another filter which uses f.server_conn.ip_address.

```
class FDst(_Rex):
    code = "dst"
    help = "Match destination address"
    is_binary = False

    def __call__(self, f):
        if not f.server_conn or not f.server_conn.address:
            return False
        r = "{}:{}".format(f.server_conn.address[0], f.server_conn.address[1])
        return f.server_conn.address and self.re.search(r)```

Reply:
I think I understand that part, but why do you want to filter by IP address and not by host name? Do you have an example where that is advantageous? :)

Reply:
As you described in the chart, I added an item “Intercept all requests from this IP”.
Sometimes, more than one host names can have the same ip address.
In such a case, it may be useful.
But if you don’t need it, I will delete it.

Reply:
Dang! Sorry, I didn't realize I actually asked for that. 🙊 I think filtering by hostname is perfectly fine here, let's go with that! 😃